### PR TITLE
filters added for unassigned bidders

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Dual Merge: [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/1324)
[Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/148)

### CODE REVIEW

File: talentmap_api/fsbid/services/client.py [hunk 1]
```diff
@@ -244,6 +244,8 @@ def fsbid_clients_to_talentmap_clients(data):
         "role_code": data.get("rl_cd", None),
         "pos_location": map_location(location),
         "hasHandshake": fsbid_handshake_to_tmap(data.get("hs_cd")),
+        "noPanel": fsbid_no_panel_to_tmap(data.get("no_successful_panel")),
+        "noBids": fsbid_no_bids_to_tmap(data.get("no_bids")),
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/client.py:248]: Are these real in fsbid? Do we also need them in the mock?

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/client.py:248]: i should confirm that, but yea, vijay was working on these and these are the ones he said he set up. Yea, i gotta set up the mock for them 👍

File: talentmap_api/fsbid/services/client.py [hunk 2]
```diff
@@ -373,6 +379,36 @@ def tmap_handshake_to_fsbid(hs):
...
+        "false": "N"
+    }
+    return tmap_dictionary.get(panel, None)
+
+def fsbid_no_bids_to_tmap(bids):
+    fsbid_dictionary = {
+        "Y": "true",
+        "N": "false"
+    }
+    return fsbid_dictionary.get(bids, None)
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/client.py:402]: Same as https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/466/files#r564611199

File: talentmap_api/fsbid/services/client.py [hunk 3]
```diff
@@ -373,6 +379,36 @@ def tmap_handshake_to_fsbid(hs):
     }
     return tmap_dictionary.get(hs, None)
 
+def fsbid_no_successful_panel_to_tmap(panel):
+    fsbid_dictionary = {
+        "Y": "true",
+        "N": "false"
+    }
+    return fsbid_dictionary.get(panel, None)
```
[the remark this answers is not in the document]
Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/client.py:387]: 🤔  true. Was following what was already there for HS, but I can go ahead and update that one too

**Comment by elizabeth-jimenez** [comment]
:(

**Comment by scott062** [comment]
Wasn't this already merged? / Approved?

**Comment by elizabeth-jimenez** [comment]
> Wasn't this already merged? / Approved?

FE was approved and merged; Mock wasn't approved, but I just merged. Mike wanted to do some testing on the passing of the true/false values 👨‍💻

File: talentmap_api/fsbid/services/client.py [hunk 4]
```diff
@@ -244,6 +244,8 @@ def fsbid_clients_to_talentmap_clients(data):
         "role_code": data.get("rl_cd", None),
         "pos_location": map_location(location),
         "hasHandshake": fsbid_handshake_to_tmap(data.get("hs_cd")),
+        "noPanel": fsbid_no_successful_panel_to_tmap(data.get("no_successful_panel")),
+        "noBids": fsbid_no_bids_to_tmap(data.get("no_bids")),
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/client.py:248]: Let's add a comment to denote that the FSBid values are not actually exposed yet

File: talentmap_api/fsbid/services/client.py [hunk 5]
```diff
@@ -269,6 +271,8 @@ def fsbid_clients_to_talentmap_clients_for_csv(data):
         "role_code": data.get("rl_cd", None),
         "pos_location": pos_location,
         "hasHandshake": fsbid_handshake_to_tmap(data.get("hs_cd")),
+        "noPanel": fsbid_no_successful_panel_to_tmap(data.get("no_successful_panel")),
+        "noBids": fsbid_no_bids_to_tmap(data.get("no_bids")),
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/client.py:275]: Let's add a comment to denote that the FSBid values are not actually exposed yet

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/client.py:275]: will do!

**Review by mjoyce91 — APPROVED** [review]

**Pull request merged**